### DISCUSSION TITLE
更新日志 - 22.03.5 - 20231020

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1063] [Announcements]
# 系统
1. 默认禁用 Docker 网络桥的 IPTables 过滤，以符合一般用户的直觉，但对于某些将 Docker 作为沙箱使用的高级用户可能增加安全风险。如果 Docker 容器的网络还有其他问题，请尝试在“网络”-“防火墙”页面的“常规设置”里，将“转发”设置为“接受”，然后保存应用

# 应用：
* iStore：
  1. 升级到 v0.1.14-2，已安装的元数据文件损坏时避免崩溃，并显示卸载按钮

# 设备
* x86：
  1. i915-oot 驱动修改为只支持11代及以上Intel CPU，并且将在12代及以上自动开启（因为默认的i915驱动不支持12代以上），采用白名单机制；如果命中白名单，开机内核日志中会出现`*** is in i915-oot white list`，如果12代以上遇到问题请反馈CPU型号或`lspci -nnk`的输出。升级固件以后，大部分用户应该都不需要再去勾选 i915-oot 驱动了，开机时将自动判断
* rk35xx：
  1. 新增支持 RK3528，第一款适配机型是 Hlink H28K
  2. 大部分机型的第一个USB口（也就是USB线刷的那个口，可能是Type-C形态）启用 OTG 支持，且支持软切换 host 或 device 模式，应用场景如模拟U盘、USB键鼠等。具体用法参考 Rockchip 的文档以及 Linux USB Gadget 相关文档，iStoreOS 中暂无相关应用

**Comment by ArnoChenFx** [comment]
在ESXI8.0中安装iStoreOS, 使用i915-oot驱动后CPU占用率会明显升高（ESXI中显示CPU占用从30%升高到110%）。CPU是N100。不知道是不是正常现象。

**Reply by jjm2473** [reply]
可能之前完全没驱动GPU？

虚拟机里面使用top命令看看进程的CPU使用率。

**Comment by blAckhAmmer6969** [comment]
你好，更新了最新的固件，外网测速只有200-500mbps/s，是电信1000mbps/s 宽带。用的是J1900，之前都能跑到900mbps，内网测速能达标。请问一下是什么原因啊？？？麻烦帮助一下

**Reply by jjm2473** [reply]
执行`lspci -nnk`看看

**Reply by rootphantomer** [reply]
我升级后接口速率协商成100M了，恢复上一个版本就是1000M。设备：R66S，目前不敢升级，观望下。

**Reply by blAckhAmmer6969** [reply]
4个网口都是1000m，砍半还好说，直接掉了三分之二的速度。我是x86，哎，希望修复一下。真的不想换固件了

**Reply by hjames1** [reply]
我也是这样，用一段时间wan口速率变成100M，重启后又回到1000M，有解决办法吗

**Reply by jjm2473** [reply]
@rootphantomer @hjames1 不是因为网线不行？

**Reply by rootphantomer** [reply]
应该没这个可能，因为网线没动，升级后下载发现速度只有9M/S多，一看协商变100M了。重启了一下我的有线ap模式的路由器发现协商不变，就直接r66s升级到上一个版本再打开首页看就正常了。

**Reply by blAckhAmmer6969** [reply]
看一下固件更新解决一下，很多人都有这个问题，掉网速。而且掉一大半！麻烦了

**Reply by jjm2473** [reply]
@blAckhAmmer6969 但是我用N100拨号可以跑满千兆啊

**Reply by jjm2473** [reply]
@rootphantomer 可能新的驱动对网络稳定性要求更高，如果网线不好就会降速。根据以往经验，只要是协商降速都是因为网线不行，换成扁网线就行了。

**Comment by KONGKAIMING** [comment]
更新了这个版本以后无法通过其他网络设备来唤醒istoreos了怎么搞?   退回20230609这个版本又能正常唤醒

**Reply by jjm2473** [reply]
这都能跟固件有关？谁没事会去改网络唤醒

**Reply by KONGKAIMING** [reply]
就是这么神奇，我为此已经升级又降级不少于五次了

**Reply by jjm2473** [reply]
bios里面设置好了吗

**Reply by KONGKAIMING** [reply]
bios设置没动过，就是系统退回来就可以了

**Comment by 160jelili** [comment]
拨号可以跑满千兆

**Reply by jjm2473** [reply]
我都测了N100，J4125，都能跑满千兆。不知道他们啥情况
